aloha-io-avro/src/test/scala/com/eharmony/aloha/factory/avro/StdAvroModelFactoryTest.scala

```diff
@@ -17,27 +17,73 @@ import org.junit.runners.BlockJUnit4ClassRunner
   */
 @RunWith(classOf[BlockJUnit4ClassRunner])
 class StdAvroModelFactoryTest {
-  @Test def testHappyPath(): Unit = {
+  import StdAvroModelFactoryTest._
 
-    val factory: ModelFactory[GenericRecord, Score] = StdAvroModelFactory(
-      "res:avro/class7.avpr",
-      "Double",
-      Seq("com.eharmony.aloha.feature.BasicFunctions._", "scala.math._")).get
+  @Test def testHappyPathExplicitUrlConfig(): Unit = {
+    val factory: ModelFactory[GenericRecord, Score] =
+      StdAvroModelFactory.fromConfig(UrlConfig(SchemaUrl, ReturnType, Imports)).get
 
-    val model: Model[GenericRecord, Score] = factory.fromString(modelJson).get
+    val model: Model[GenericRecord, Score] = factory.fromString(ModelJson).get
+
+    assertEquals(7d, model(record).getValue.asInstanceOf[Double], 0)
+  }
+
+  @Test def testHappyPathUrlMagnetSyntax(): Unit = {
+    // Must import syntax explicitly to use Magnet syntax. Can do
+    // import FactoryConfig.MagnetSyntax._
+    import FactoryConfig.MagnetSyntax.fromUrlCodomainImports
```

Comment:
Should the Magnet syntax be automatic (defined in the implicit scope of FactoryConfig) or should we require explicit imports to enable the syntax?  Or should the syntax be removed entirely as it's not necessary?  The more I think about it, the more I think it should be removed and added later if we choose.  A lot easier to add than remove.